package.json

```diff
@@ -6,12 +6,6 @@
 	"private": true,
 	"scripts": {
 		"postinstall": "./scripts/postinstall.sh",
-		"storybook": "pnpm '/^storybook:.*/'",
```

Comment:
how would you run both storybooks at the same time now? or is not something people ever do?